### DISCUSSION TITLE
Committing JSON text changes

### DISCUSSION THREAD

**Discussion by are0h** [obsproject/obs-studio#7086] [Development Questions] [answered]
What's up, all. I had a simple question about updating some URLs in the `rtmp-services/data/services.json` file. Some URLs for Switchboard Live have been deprecated and need to be changed. 

I've read the contribution guide at that's fine, but I wanted to ask if there was another process for updating a few lines of text that won't trigger all the notification bells and wells. 

If not, and I should just follow the guide, that's cool. Just checking in to see if there is a streamlined path for simple stuff.

**Comment by sebastian-s-beckmann** [comment, marked as answer]
There's a service-specific guide at https://github.com/obsproject/obs-studio/wiki/Service-Submission-Guidelines, you might be looking for that.
Most of it is for new services but it generally also applies to updates to existing ones.

**Reply by are0h** [reply]
Much appreciated. I'll give it a look.
